frontend/components/employee/index.js

```diff
@@ -0,0 +1,108 @@
+import SearchInput from "../service/search";
+import * as React from "react";
+import Box from "@mui/material/Box";
+import {Divider, Grid,} from "@mui/material";
+import Button from "@mui/material/Button";
+import {useState} from "react";
+import styled from '../../styles/employee.module.css';
+import NewEmployeeDialog from "./newEmployeeDialog";
+import {useRouter} from "next/router";
+import {DataGrid} from "@mui/x-data-grid";
+
+
+const EmployeeComponent = (props) => {
+    const router = useRouter();
+    const {employeeList, addEmployee, deleteEmployee, serviceList} = props;
+    const [displayEmployeeList, setDisplayEmployeeList] = useState(employeeList);
+    const [addOpen, setAddOpen] = useState(false);
+    const [rowSelection, setRowSelection] = useState([]);
+
+    const closeAddOpen = () => {
+        setAddOpen(false);
+    };
+
+
+    const handleSearch = (val) => {
+        let searchValue = val.toLowerCase().trim()
+        if (searchValue.length > 0) {
+            let temp = employeeList.filter(
```

Comment:
I would rename this variable and use `const` here.

Reply:
ok, done